lib/SPIRV/libSPIRV/SPIRVType.h

```diff
@@ -656,20 +665,59 @@ class SPIRVTypeStruct : public SPIRVType {
     MemberTypeIdVec.resize(WordCount - 2);
   }
 
+  // TODO: Should we attach operands of continued instructions as well?
   std::vector<SPIRVEntry *> getNonLiteralOperands() const override {
     std::vector<SPIRVEntry *> Operands(MemberTypeIdVec.size());
     for (size_t I = 0, E = MemberTypeIdVec.size(); I < E; ++I)
       Operands[I] = getEntry(MemberTypeIdVec[I]);
     return Operands;
   }
+  void addContinuedInstruction(ContinuedInstType Inst) {
+    ContinuedInstructions.push_back(Inst);
+  }
+
+  void encodeChildren(spv_ostream &O) const override {
+    O << SPIRVNL();
+    for (auto &I : ContinuedInstructions)
+      O << *I;
+  }
+
+  // TODO: return a reference here?
```

Comment:
Compiler should take care of this automatically: [copy elision](https://en.cppreference.com/w/cpp/language/copy_elision)

Reply:
Ok. Removed TODO comment.